client/src/components/BottomBanner/index.tsx

```diff
@@ -39,6 +39,15 @@ const BottomBanner = memo(
       });
     };
 
+    const bannerContent: JSX.Element = (
```

Comment:
minor optimization lol

Maybe we can hoist this constant content out of the component, so we don't recreate a new one at every render 😄 ?